src/router/helpers.ts

```diff
@@ -2,7 +2,7 @@ import {Route} from 'vue-router'
 
 export const scrollBehavior = (to: Route) => {
   if (to.params.field) {
-    const anchor = `block/${to.params.blockId}${to.params.field}`
+    const anchor = to.params.field
```

Comment:
Why this changes ? this may affect line like https://github.com/FLOIP/flow-builder/pull/221/files#diff-9754383fb88bba0a3f438d1811b59308b89ee44c5c5c5d202b0409d83b8a5e89R16